cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -197,6 +210,16 @@ func StartTestServer(t Logger, instanceOptions *TestServerInstanceOptions, custo
 		}
 	}(stopCh)
 
+	if instanceOptions.SkipPostStartChecks {
```

Comment:
The calls below look to be related to reads, not writes.  What causes us to need to skip the poststart checks?

Reply:
Since we wrapped the storage version poststart hook for testing, the /healthz won't become ok. This allows us to skip waiting for /healthz to return ok when bootstrapping the test server. Also the default namespace creation check gets skipped since it happens after the healthz check.

Reply:
Made the if block capture the entire healthz check & namespace check section.